### ISSUE TITLE
Application List not visible via Tizen Studio

### ISSUE THREAD

**Issue by TiPunch69** [tavicu/homebridge-samsung-tizen#378]
I installed Tizen Studio (Version 4.1.1) with the TV extension and conneted the TV. All works fine, nevertheless the List of Installed WebApps is nowhere to be found in the dropdown of the device manager.

**Comment by tavicu** [comment]
Hi, sorry but i cannot help you with this since i don't know more about Tizen Studio.

What i can tell you is that i had the same problem and after looking better, i had installed a wrong extension from the list.

**Comment by TiPunch69** [comment]
Which extension did you finally install?
I clicked on the "Extension SDK" Tab in the Package Manager and on the bottom on "TV Extension Tools"

**Comment by TiPunch69** [comment]
> Hi, sorry but i cannot help you with this since i don't know more about Tizen Studio.
> 
> What i can tell you is that i had the same problem and after looking better, i had installed a wrong extension from the list.

Your hint did the trick. You need to install "**TV Extension Tools-6.0**" not "TV Extension Tools" from the bottom of the list. I think this should be mentioned in the configuration documentation.

**Comment by tavicu** [comment]
Yup, in the Samsung Developers doc it say TV Extensions-<version>

I have updated the wiki to say `TV Extensions-<version>`

**Comment by TiPunch69** [comment]
Thanks, issue can be closed. Cool plugin BTW, thanks for the work.
